docs/learn-ledger.md

```diff
@@ -0,0 +1,166 @@
+---
+id: learn-ledger
+title: Using the Polkadot Ledger Application
+sidebar_label: Ledger Application
+---
+
+The Polkadot [Ledger][] application is compatible with both the Ledger Nano S and the Ledger Nano X
+devices. Ledger devices are hardware wallets that keep your secret secured on a physical device that
+does not expose it to your computer or the internet. It is strongly recommend to use a hardware
+wallet if you are managing a significant amount of funds.
+
+The Polkadot application allows you to manage Polkadot's native currency, the DOT. It supports most
+of the transaction types of the network (a notable exception is the "Batch" transaction from the
+Utilty pallet).
+
+## Requirements
+
+Here is a list of what you will need before starting:
+
+- A Ledger Nano S or a Ledger Nano X.
+- The latest firmware installed. (At the time of writing this is 1.6)
+- Ledger Live is installed.
+- A web browser is installed that you can use to access [Polkadot-JS Apps][].
+
+## Installing the Ledger Application
+
+### Using Ledger Live
+
+- Open the "Manager" tab in Ledger Live.
+- Connect and unlock your Ledger device.
+- If asked, allow the manager on your device by pressing the right button.
+- Find Polkadot in the app catalog.
+- Click the "Install" button of the app.
+- An installation window appears.
+- Your device will display "Processing…"
+- The app installation is confirmed.
+
+Please proceed to the usage instruction below.
+
+### Using the Developer Release
```

Comment:
Does it make sense to keep the Developer release installation instructions after it goes live?

Reply:
I'll remove it in a follow-up PR when the application is live. I'm planning to take some screenshots for the Ledger Live installation steps and will remove the dev installation instructions then.